api/tests/test_archive_querier.py

```diff
@@ -249,10 +249,12 @@ def get_multiple_pages(query_function, query_args):
 def get_all_pages(query_function, query_args):
     pages = []
     cursor = None
+    counter = 1
     while True:
         page = get_page(query_function, query_args, cursor)
         pages.append(page)
         cursor = page.cursor
+        counter += 1
```

Comment:
hmm am i missing where this counter is used? I don't see why its added.

Reply:
Derp. Also a debug artifact. Thanks.